copier/main.py

```diff
@@ -337,16 +337,21 @@ def answers(self) -> AnswersMap:
             )
         for question in questions:
             # Display TUI and ask user interactively only without --defaults
-            new_answer = (
-                question.get_default()
-                if self.defaults
-                else unsafe_prompt(
-                    question.get_questionary_structure(), answers=result.combined
-                )[question.var_name]
-            )
+            try:
+                new_answer = (
+                    question.get_default()
+                    if self.defaults
+                    else unsafe_prompt(
+                        question.get_questionary_structure(), answers=result.combined
+                    )[question.var_name]
+                )
+            except KeyboardInterrupt:
+                raise CopierAnswersInterrupt(result, question, self.template)
```

Comment:
```suggestion
                raise CopierAnswersInterrupt(result, question, self.template) from err
```

AFAIK and According to docs this is the correct way of converting one exception into another.